packages/ses/src/compartment-evaluate.js

```diff
@@ -0,0 +1,72 @@
+/// <reference types="ses">
```

Comment:
The `Compartment.evaluate` implementation needed to be extracted into a stand-alone module so it could be used from both `compartment-shim.js` and `module-instance.js` without dynamic dispatch. The internals communicate purely through their captured private fields.